project_index.php

```diff
@@ -0,0 +1,20 @@
+<?php
+require_once('functions.php');
+$db = databaseConnect();
+$characters = databaseFetchAll($db);
+
+?>
+
+<html>
+    <head>
+        <!-- Create a navbar that will allow access to creation/editing/deleting -->
+    </head>
+    <body>
+        <div>
+            <?=displayCharacters($characters);?>
+        </div>
+        <form action='char_creation.php'>
+            <button>Create a new character</button>
+        </form>
```

Comment:
Rather than using a form here to link to the char_creation.php page, you should just use an a tag.

While the form does work and doesn't do anything bad, it's just 'overkill' for what this button is actually doing. It does mean you'll need to write some CSS for the a tag if you want it to look button like